libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.h

```diff
@@ -285,6 +290,24 @@ extern NSString* const kSFRestDefaultAPIVersion;
                                            fields:(nullable NSDictionary<NSString*, id> *)fields;
 
 /**
+ * Same as requestForUpdateWithObjectType:objectId:fields but only executing update
+ * if the server record was not modified since ifModifiedSinceDate.
+ *
+ * @param objectType object type; for example, "Account"
+ * @param objectId the record's object ID
+ * @param fields an object containing initial field names and values for record
+ * @param ifUnmodifiedSinceDate update will only happens if current last modified date of record is
+ *                              older than ifUnmodifiedSinceDate
+ *                              otherwise a 412 (precondition failed) will be returned
+ * @see http://www.salesforce.com/us/developer/docs/api_rest/Content/resources_sobject_retrieve.htm
+ */
+- (SFRestRequest *)requestForUpdateWithObjectType:(NSString *)objectType
+                                         objectId:(NSString *)objectId
+                                            fields:(nullable NSDictionary<NSString*, id> *)fields
+                            ifUnmodifiedSinceDate:(nullable NSDate *) ifUnmodifiedSinceDate;
```

Comment:
Oh, I see why it's nullable: because it's a default case for the method above.  Makes sense.